test-resources/headless-core-stub/lambdas/callback/src/callback-handler.ts

```diff
@@ -1,12 +1,87 @@
-import type { LambdaInterface } from "@aws-lambda-powertools/commons/types";
+import { LambdaInterface } from "@aws-lambda-powertools/commons/types";
+import { APIGatewayProxyEvent, APIGatewayProxyResult, Context } from "aws-lambda";
+import { Logger } from "@aws-lambda-powertools/logger";
+import { ConfigurationHelper } from "./services/configuration-helper";
+import { CallBackService } from "./services/callback-service";
+import { generatePrivateJwtParams } from "./services/private-key-jwt-helper";
 
+const sessionTableName = process.env.SESSION_TABLE || "session-common-cri-api";
+const configurationHelper = new ConfigurationHelper();
+const callbackService = new CallBackService();
+const logger = new Logger();
 export class CallbackLambdaHandler implements LambdaInterface {
-    async handler(): Promise<{ status: string; body: string }> {
-        return {
-            status: "200",
-            body: "Hello callback",
-        };
+    async handler(event: APIGatewayProxyEvent, _context: Context): Promise<APIGatewayProxyResult> {
+        try {
+            const authorizationCode = event.queryStringParameters?.authorizationCode;
+
+            if (!authorizationCode) {
```

Comment:
Is this possible with the API GW config that we have?

Reply:
Ok nice, yeah I would be in favor of us removing now that we are done debugging as API GW should give us a very similar errros